net-im/signal-desktop-bin/signal-desktop-bin-1.11.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit gnome2-utils pax-utils unpacker xdg-utils
+DESCRIPTION="Allows you to send and receive messages of Signal Messenger on your computer"
+HOMEPAGE="https://signal.org/ https://github.com/WhisperSystems/Signal-Desktop"
+SRC_URI="https://updates.signal.org/desktop/apt/pool/main/s/${MY_PN}/${MY_PN}_${PV}_amd64.deb"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="ayatana"
```

Comment:
why did you add this flag? it does not affect the installation, so install it unconditionally

Reply:
See here: https://bugs.gentoo.org/645814

Reply:
noted, seems like false positive statement for me

Reply:
@Zlogene I removed the flag as you suggested.